radlab-ui/webapp/src/components/forms/DefaultStepCreator.tsx

```diff
@@ -47,6 +47,9 @@ const DefaultStepCreator: React.FC<IDefaultStepCreatorProps> = ({
       }
     })
     setZoneListData(zoneListData)
+
+    // set first value as default zone while region changed
+    values["zone"] = zoneListData[0]
```

Comment:
I don't love doing a side effect here. I'd prefer if this function got renamed to `getZoneByRegion` and have it return that string value. And then invoke that function in `onChangeRegion`